### ISSUE TITLE
Play Value not working

### ISSUE THREAD

**Issue by Sailor-Boy** [FrSkyRC/ETHOS-Feedback-Community#786]
I have updated to the newest version 1.1.10
I cannot get any telemetry values to announce under Play Value, 
I have tried Voltage, RSSI, RX-voltage but none will announce.
Play track works fine.
Am I missing something?

**Comment by m0p3d** [comment]
Define Logic Switch and Setup when to activate.
Create Special Function trigger to play based on previous defined Logic Switch status.

@bsongis-frsky Do you plan to have any default option to activate Play Varaible Name before "Value"
For example if u setup something based on fuel Level and Trigger <50% -> ATM you get Play "50%" - would be nice to be able to get "Fuel: 50%"

**Comment by bsongis-frsky** [comment]
Will be possible with #441

**Comment by Sailor-Boy** [comment]
I did setup a logic switch to activate below a threshold voltage (if lipo voltage drops below 15 volts logic switch activates) a test of this function works just fine.  I then created a special function to play a Telemetry Value - Lipo that triggers based on the Logic switch.  Everything works but the value is not announced.  I swapped out the "Play Value" for a "Play Track" .  Everything works fine.  Voltage drops below 15 volts, logic S/W triggers, special function plays the track, all good.  Switch back to "Play Value" (tried all telemetry values, RSSI, RX Voltage, Lipo Voltage) ....nothing.
Does there need to be some sort of default audio file on the SD card in order for the voice to announce the telemetry values?
